rewrite-java/src/main/java/org/openrewrite/java/cleanup/UselessCompound.java

```diff
@@ -0,0 +1,22 @@
+package org.openrewrite.java.cleanup;
+
+import org.openrewrite.ExecutionContext;
+import org.openrewrite.Recipe;
+import org.openrewrite.TreeVisitor;
+
+public class UselessCompound extends Recipe {
```

Comment:
The recipe name may be interpreted as a useless compound, I think this would read better as something like, `SimplyCompoundStatements`.